packages/backend-common/package.json

```diff
@@ -151,5 +151,6 @@
       "optional": true
     }
   },
-  "configSchema": "config.d.ts"
+  "configSchema": "config.d.ts",
+  "deprecated": true
```

Comment:
Should I revert this change and do this instead?
https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

Reply:
yep let's do that

Reply:
Done 🙏🏻